section1/reflection.md

```diff
@@ -1,19 +1,39 @@
+    - Booleans have to be either truthy or falsy. They use operators like ===, !==, <, or <= to compare two values and return true or false.
 
 3. How would you log the string `"Hello World!"` to the console?
+  console.log('Hello World!');
 
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+  Code comments can describe a block of code or provide more context. They will not be shown when the code is run.
+  A single line comment is designated with two // This is a comment!
+  A multi-line comment is designated as follows /* This is a comment! */
 
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
+  A variable is another was to rename and store data. You can assign a longer piece of information a shorter name, but it will equal the same thing.
+
+  If I was describing this to a 5 year old, I think I would use the metaphor of having a nickname. For example, my family calls me Liv, but it doesn't change the fact that my name is Olivia and they know that my name is Olivia. When they say 'Liv' everyone knows who they are talking about because this nickname has been 'assigned' to me!
 
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+  I use Spotify every day. I would guess that the following three variables are used: 'nextSong', 'playlistName', and 'userName'. The nextSong variable could hold the next song that I've queued up, the playlistName could represent the title that I have given a playlist of mine, and the userName could hold my specific Spotify name.
 
 7. In your own words, explain what concatenation is.
+  Concatenation is one method of combining strings, numbers, and variables together so that correctly print to the console. It is not as streamlined as interpolation, and it can be a little harder to read. Data types and variables are combined using a combination of quotation marks and + signs!
 
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+  Again, I'll go back to Spotify. When I open the app to listen to music, I get a message that says something like "Welcome back Olivia!". Although it's a simple message, they could have used concatenation to make it. I would write it out as follows:
+      userName = "Olivia";
+      console.log("Welcome back " + userName + "!");
 
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+  I am a bit overwhelmed at how everything eventually interacts together! I feel like the simple act of making a Git repository and linking local/remote repositories together takes all of my concentration and memory. Repetition and practice is already helping, but the scale of it all is crazy to me. I don't understand how so many things interact at once.
```

Comment:
How do you feel about these two things you brought up, now that Mod 0 is done and you've completed the project?!